audit-triplestore/src/test/java/org/fcrepo/camel/audit/triplestore/RouteTest.java

```diff
@@ -59,7 +59,7 @@ public void testEventProcessor() throws Exception {
         final Map<String, Object> headers = new HashMap<>();
         headers.put(JmsHeaders.BASE_URL, "http://localhost/rest");
         headers.put(JmsHeaders.IDENTIFIER, "/foo");
-        headers.put(JmsHeaders.TIMESTAMP, "1428360320168");
+        headers.put(JmsHeaders.TIMESTAMP, 1428360320168L);
```

Comment:
Does the JMS message from fedora have this value serialized as a Long? Correct me if I'm wrong, but I thought everything comes across the wire as a String.